projects/jsoncpp/build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -eu
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+mkdir -p $WORK/jsoncpp
+cd $WORK/jsoncpp
+
+CFLAGS="$CFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE"
+CXXFLAGS="$CXXFLAGS -DBORINGSSL_UNSAFE_FUZZER_MODE"
+
+cmake -D CMAKE_BUILD_TYPE=Release \
+    -DCMAKE_C_COMPILER="$CC" -DCMAKE_CXX_COMPILER="$CXX" \
+    -DCMAKE_C_FLAGS="$CFLAGS" -DCMAKE_CXX_FLAGS="$CXXFLAGS" \
+    -Wno-dev $SRC/jsoncpp/
+
+make install "-j$(nproc)"
+
+$CXX $CXXFLAGS -std=c++11 -I.. $SRC/jsoncpp/src/test_lib_json/fuzz.cpp \
+ -lFuzzingEngine ./src/lib_json/libjsoncpp.a -o $OUT/jsoncpp_fuzzer
```

Comment:
Please use `$LIB_FUZZING_ENGINE` instead of `-lFuzzingEngine`

Reply:
Ok, got it.